crates/volta-core/src/path/mod.rs

```diff
@@ -44,6 +45,10 @@ pub fn ensure_volta_dirs_exist() -> Fallible<()> {
         ensure_dir_exists(user_toolchain_dir()?)?;
         ensure_dir_exists(tmp_dir()?)?;
         ensure_dir_exists(log_dir()?)?;
+        // also ensure the basic shims exist
+        ensure_shim_exists("node")?;
+        ensure_shim_exists("yarn")?;
+        ensure_shim_exists("npm")?;
```

Comment:
For consistency with the installer, we should ensure the `npx` shim exists as well.

Reply:
doh! forgot about that one - will add it